README.md

```
@@ -16,9 +16,9 @@ Other useful resources:
 
 - [Download and Install Mattermost](https://docs.mattermost.com/guides/deployment.html) - Install, setup, and configure your own Mattermost instance.
 - [Product documentation](https://docs.mattermost.com/) - Learn how to run a Mattermost instance and take advantage of all the features.
-- [Developer documentation](https://developers.mattermost.com/) - Contribute code to Mattermost or build an integration via APIs, Webhooks, slash commands, Apps, and plugins.
+- [Developer documentation](https://developers.mattermost.com/) - Contribute code to Mattermost or build an integration via APIs, webhooks, slash commands, Apps, and plugins.
 
-Table of contents
+Table of Contents
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)